packages/read-model/src/Product/Listed/ListedProductViewFacade.php

```diff
@@ -205,46 +203,23 @@ public function getFilteredPaginatedForSearch(string $searchText, ProductFilterD
     public function getPaginatedForBrand(int $brandId, string $orderingModeId, int $page, int $limit): PaginationResult
     {
         $paginationResult = $this->productOnCurrentDomainFacade->getPaginatedProductsForBrand($orderingModeId, $page, $limit, $brandId);
-
-        return $this->createPaginationResultWithArray($paginationResult);
+        return $this->createPaginationResultWithData($paginationResult);
     }
 
     /**
      * @param \Shopsys\FrameworkBundle\Component\Paginator\PaginationResult $paginationResult
      * @return \Shopsys\FrameworkBundle\Component\Paginator\PaginationResult
      */
-    protected function createPaginationResultWithArray(PaginationResult $paginationResult): PaginationResult
+    protected function createPaginationResultWithData(PaginationResult $paginationResult): PaginationResult
```

Comment:
this name hits me again... maybe just `createPaginationResult`?

Reply:
This name was already in read model and I didn't know how to rename it. just `createPaginationResult` seems more confusing to me

Reply:
if so, ok